dali/base/dafdesc.cpp

```diff
@@ -37,9 +37,14 @@
 
 #define SDS_CONNECT_TIMEOUT  (1000*60*60*2)     // better than infinite
 
+// These are legacy and cannot be changed.
 #define SERIALIZATION_VERSION ((byte)0xd4)
 #define SERIALIZATION_VERSION2 ((byte)0xd5) // with trailing superfile info
 
+#define VERSION_MINHISTORY 1  // Add History to serialisation/de-serialisation
+// This version serialised in the extendedVersion of CFileDescriptor
+static const unsigned short serializationVersion = VERSION_MINHISTORY;
```

Comment:
trivial: but you expect the version to increment as time goes on - it's a coincidence it's equal to the min version of the last feature added, so looks little odd to say this.
Better to just say = 1 here.